### ISSUE TITLE
Grid (from file: tif or asc) is not projecting correctly

### ISSUE THREAD

**Issue by ThomasOsinga** [gama-platform/gama.old#3859]
**Describe the bug**
I am working on a model where my background environment is a grid from a raster file. I am also working with multiple shapefiles. In QGIS/ArcGis and R these files are all the correct CRS and overlap perfectly. When I import them into gama and run my experiment I get the following error: 

 at cycle 0: Try to project a grid -../includes/Suitability.tif-  that is not projected. Projection of grids can lead to errors in the cell coordinates.
in Simulation 0
in

And then the grid is 20 cells shifted towards the right compared to my shapefiles. I have no idea how to fix this, can anyone help me? 

**To Reproduce**
Run the attached model 

**Expected behavior**
I would expect my shapefiles and grid to be alligned correctly (you can see the purple/pink part that should perfectly overlap with my shapefile of that area. 

**Screenshots**
![image](https://github.com/gama-platform/gama/assets/86662587/e4db48e6-252f-4f79-9577-9c98c1c8e73d)

**Desktop (please complete the following information):**
 - OS: Windows
 - PC Model: different PC's, same issue. 
 - GAMA version: 1.9.1
 - Java version: Version 8 update 361 - b09
 - Graphics cards / Display system:  //

**Additional context**

Shifting the raster in GIS and then importing it in GAMA changed nothing - Changing the Shape geometry changes how much off-set it is (for example using a different raster with the same extent changed how off the other raster was). 

I added dummy code to reproduce the issue. 

[Testing_projection.zip](https://github.com/gama-platform/gama/files/12498082/Testing_projection.zip)

**Comment by ThomasOsinga** [comment]
I have been trying to fix this for 3 days, and now that I made a ticket I fixed it within 30 minutes... 

I still don't know what caused it, but creating a new file in a different projection system (UTM instead of RD_new, both projected projection systems) worked! I still do not understand it as I have tried with like 5 different projections before this one for all files. 

But this WGS84 UTM-N31 seems to work. Even stranger is that it even works while the veluwe shapefile is RD_new or UTM-N31 (it makes no difference).
